test/functional/multiwallet.py

```diff
@@ -15,23 +20,49 @@ def __init__(self):
         self.extra_args = [['-wallet=w1', '-wallet=w2', '-wallet=w3']]
 
     def run_test(self):
+        self.stop_node(0)
+
+        # should not initialize if there are duplicate wallets
+        self.assert_start_raises_init_error(0, self.options.tmpdir, ['-wallet=w1', '-wallet=w1'], 'Error loading wallet w1. Duplicate -wallet filename specified.')
```

Comment:
missing 10556

Reply:
added, but LOTS of conflicts so likely some errors